WasatchNET/AndorEEPROM.cs

```diff
@@ -85,9 +85,14 @@ public override bool read()
             calibrationDate = "01/01/2020";
             calibrationBy = "RSC";
 
-            detectorName = "";
+            int cameraSerial = 0;
+            uint error = andorDriver.GetCameraSerialNumber(ref cameraSerial);
+            if (error != AndorSpectrometer.DRV_SUCCESS)
+                detectorName = "";
+            else
+                detectorName = cameraSerial.ToString();
```

Comment:
Right now, ENLIGHTEN is using "iDus" for eeprom.detector.  The camera serial is just a 5-digit number, like 27239.  To me, this could get confusing if we encountered a camera serial number like 16011, 11850, 14672 etc.

Reply:
there have been discussions of wanting to track the camera's serial number with WNET and to me this made sense as a spot for it. We can pick something better as a group though